### ISSUE TITLE
DB - don't do DNS check on IPs

### ISSUE THREAD

**Issue by Andi-K** [friendica/friendica#4786]
### Expected behavior
A working installation procedure when entering an IP as DB server. 

### Actual behavior
You can't continue to install Friendica and get the wrong error "Connection to the database failed.".

### Steps to reproduce the problem
Try to install Friendica using a valid IP:Port as DB Server

### Friendica version you encountered the problem
since 2.1.97
Yeah! 7 years old and no one has tried/reported it?

### source of error 
commit 0cf647bf6f199586bad2e13973fe034bf5676eea

dns_get_record() expects a hostname - no IP - and fails
The notice 'Cannot locate DNS info for database server' was never displayed.

After removing this check, all went well.

**Comment by Andi-K** [comment]
IMO this check is unnecessary.

**Comment by MrPetovan** [comment]
Thanks for tracing back the original commit!

**Comment by M-arcus** [comment]
This is the same issue I had, after removing the constant `DNS_PTR` from the function the dns check worked again.

**Comment by M-arcus** [comment]
I would suggest adding a check if the hostname really is a hostname and not an IP, right?

**Comment by Andi-K** [comment]
Or we remove it completely. Maybe it's outdated and solved by other code change?

I think I had no problems installing Master without checking, even without checking and if an incorrect server was specified.  I just got the error message that no contact to the DB was possible. 
During my dev installation, which I'm trying to get running, I got a WSOD. I don't know why yet.

However, I find it messy in general. The check is part of the installation and should therefore be done in the install.php. If there is really need for it.
